.github/workflows/docker-image.yml

```diff
@@ -70,10 +82,15 @@ jobs:
           docker images
     - name: Login to Docker
       if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'
-      uses: docker/login-action@v1
+      uses: docker/login-action@v2
       with:
         username: ${{ secrets.DOCKERHUB_USER }}
         password: ${{ secrets.DOCKERHUB_PASS }}
-    - name: Push docker image
+    - name: Build and push docker image
       if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/master'
-      run: docker-compose push
+      uses: docker/bake-action@v2
+      with:
+        files: docker-compose.yml
+        push: true
+        set: |
+          *.platform=linux/amd64,linux/arm64
```

Comment:
Because there is an if statement, it runs only for tags and for the master branch.